### DISCUSSION TITLE
How to add Folders to the left sidebar?

### DISCUSSION THREAD

**Discussion by ketonik** [spacedriveapp/spacedrive#2401] [Community Support]
Got me stumped.
I can't seem to drag a folder to the  sidebar, there's no right click on folder, add to sidebar or add to favourites.
Need to have frequently accessed folders in the sidebar.
Probably nice if we could hide some segments of sidebar too, like Tags and Locations, because already have all SSD's listed under Local.
![2024-04-26 06_12_45-Spacedrive](https://github.com/spacedriveapp/spacedrive/assets/49221849/1d9284f4-83c9-4633-a582-96c8004bd725)

**Comment by utkubakir** [comment]
We have big plans for sidebar customization, favourite folders and drag to order etc. are definitely one of them.
